### ISSUE TITLE
left quote top aligned; should be center aligned

### ISSUE THREAD

**Issue by engerm** [L4GG/timeline#42]
The left quote is top aligned. All other left quotes or images are center aligned in order to be consistent.

See screen shot

http://timeline.lawyersforgoodgovernment.org/#event-2016-release-of-russian-collected-information-about-trump

#### Event Title
Release of Russian Collected Information About Trump

#### Event Date
06-01-2016

**Comment by engerm** [comment]
<img width="1155" alt="screen shot 2017-05-10 at 11 51 36 pm" src="https://cloud.githubusercontent.com/assets/17053096/25931709/ffa4df2c-35db-11e7-806b-373475561fe8.png">

**Comment by bernardwang** [comment]
Quick PR [here #51](https://github.com/L4GG/timeline/pull/51)
Trying to fix checks, but doesn't look like my PR introduced them...

**Comment by anseljh** [comment]
Yeah, the thing with the checks complaining about 302 redirects is a known issue. You can ignore them.
